test/DSRAuthOracle.t.sol

```diff
@@ -33,6 +32,10 @@ contract DSRAuthOracleTest is Test {
         skip(1 * (365 days));
     }
 
+    function test_constructor() public {
+        assertEq(oracle.maxDSR(), 0);
+    }
+
```

Comment:
Need full testing for `setMaxDSR`:
- Permissioning
- `< RAY` boundary condition
- `>= RAY` condition
- 0 condition

Reply:
added